Compile error No such file or directory  #include "sqlite3.h"

On my Ubuntu 18.04 box (gcc 7.4.0, cmake 3.10.2) I get this compile error:
```
[ 95%] Building CXX object tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/sqlite_transfer.cpp.o
In file included from /home/sn/code/duckdb/tools/dbtransfer/sqlite_transfer.cpp:1:0:
/home/sn/code/duckdb/tools/dbtransfer/include/sqlite_transfer.hpp:2:10: fatal error: sqlite3.h: No such file or directory
 #include "sqlite3.h"
          ^~~~~~~~~~~
compilation terminated.
tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/build.make:62: recipe for target 'tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/sqlite_transfer.cpp.o' failed
make[3]: *** [tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/sqlite_transfer.cpp.o] Error 1
make[3]: Leaving directory '/home/sn/code/duckdb/build/release'
CMakeFiles/Makefile2:7942: recipe for target 'tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/all' failed
make[2]: *** [tools/dbtransfer/CMakeFiles/sqlite_transfer.dir/all] Error 2
make[2]: Leaving directory '/home/sn/code/duckdb/build/release'
Makefile:83: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/home/sn/code/duckdb/build/release'
Makefile:24: recipe for target 'release' failed
make: *** [release] Error 2
```

It seems like the sqlite header provided under third_party is not found (I have no libsqlite3 installed).

--- a/benchmark/dbwrapper/CMakeLists.txt
+++ b/benchmark/dbwrapper/CMakeLists.txt
@@ -1,3 +1,3 @@
-
+include_directories(../../third_party/sqlite/include)
 add_library(duckdb_benchmark_dbwrappers OBJECT sqlite_benchmark.cpp)
 set(BENCHMARK_OBJECT_FILES ${BENCHMARK_OBJECT_FILES} $<TARGET_OBJECTS:duckdb_benchmark_dbwrappers> PARENT_SCOPE)

--- a/benchmark/micro/CMakeLists.txt
+++ b/benchmark/micro/CMakeLists.txt
@@ -1,3 +1,3 @@
-
+include_directories(../../third_party/sqlite/include)
 add_library(duckdb_benchmark_micro OBJECT groupby.cpp in.cpp multiplications.cpp orderby.cpp pointquery.cpp indexcreation.cpp rangejoin.cpp rangequery.cpp window.cpp)
 set(BENCHMARK_OBJECT_FILES ${BENCHMARK_OBJECT_FILES} $<TARGET_OBJECTS:duckdb_benchmark_micro> PARENT_SCOPE)

--- a/benchmark/tpcds/CMakeLists.txt
+++ b/benchmark/tpcds/CMakeLists.txt
@@ -1,3 +1,4 @@
 include_directories(../../third_party/dsdgen/include)
+include_directories(../../third_party/sqlite/include)
 add_library(duckdb_benchmark_tpcds OBJECT sf1.cpp)
 set(BENCHMARK_OBJECT_FILES ${BENCHMARK_OBJECT_FILES} $<TARGET_OBJECTS:duckdb_benchmark_tpcds> PARENT_SCOPE)

--- a/benchmark/tpch/CMakeLists.txt
+++ b/benchmark/tpch/CMakeLists.txt
@@ -1,3 +1,4 @@
 include_directories(../../third_party/dbgen/include)
+include_directories(../../third_party/sqlite/include)
 add_library(duckdb_benchmark_tpch OBJECT sf1.cpp read_lineitem.cpp)
 set(BENCHMARK_OBJECT_FILES ${BENCHMARK_OBJECT_FILES} $<TARGET_OBJECTS:duckdb_benchmark_tpch> PARENT_SCOPE)

--- a/tools/dbtransfer/CMakeLists.txt
+++ b/tools/dbtransfer/CMakeLists.txt
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 2.8.7)
 project(dbtransfer CXX C)
 
 include_directories(include)
+include_directories(../../third_party/sqlite/include)
 
 add_library(sqlite_transfer STATIC sqlite_transfer.cpp)
 target_link_libraries(sqlite_transfer duckdb sqlite3)

